### ISSUE TITLE
Add-on Crashes

### ISSUE THREAD

**Issue by espidus** [egtwobits/mesh_mesh_align_plus#13]
Windows 10 version 1903 ( OS Build 18362.900 )
Tested on Blender 2.80 and 2.82

Using "Quick Align Planes"-
1) I uncheck "Auto Grab Source..."
2) I choose Source Pt. A
3) Blender crashes out of program back onto desktop

Does not happen if using "Auto Grab Source..."

**Comment by egtwobits** [comment]
What version of Mesh Align Plus? I tried this with Mesh Align Plus v0.6.0 on Blender 2.83 just now and could not reproduce this (more info would be helpful).

**Comment by espidus** [comment]
I will check, but it I just downloaded it off of your site about 3 hours ago, so I'm assuming it's the latest version.

**Comment by egtwobits** [comment]
Make sure NOT to use "clone or download", check the wiki/installation instructions page.

**Comment by espidus** [comment]
I had downloaded it from that link and extracted it and used the zip inside the zip. I was not able to fund a link to any other. Now that I am on my mobile phone I see a link that takes me to the page with the "assets" link at the bottom with what I assume is the correct zip. If I set my mobile web browser to use the "desktop site" then the link on the main page is not there and I can't see a way to get to that other page. I will copy the link on mobile and send it to my desktop so I could download the correct version, and I will test that. Thanks for your reply.
